### ISSUE TITLE
When manually scanning for alt text or tags, if request fails, spinner never goes away

### ISSUE THREAD

**Issue by dkotter** [10up/classifai#230]
**Describe the bug**
Currently, if you use the manual scan/rescan buttons for image alt text and/or tags, as that process is working, the button is disabled and a spinner is shown. If that request completes successfully, that spinner is removed and the button is active again.

But if something goes wrong in that request, we currently don't handle that error. So the button stays disabled and the spinner stays showing, making it seem like things are still processing even though they aren't.

**Steps to Reproduce**
1. Setup the plugin
2. Disable either the Automatically Caption Images or Automatically Tag Images options (or both)
3. Select an existing image in the media library or upload a new one
4. With the image selected, in the media modal that opens, click the button that corresponds to the setting disabled above
5. See the spinner show and never go away

**Expected behavior**
If a request fails, at the very least, we should remove the loading state so the user knows things are done. Ideally, we would also show an error message.
